ui/app/serializers/metrics.js

```diff
@@ -0,0 +1,16 @@
+import ApplicationSerializer from './application';
+
+export default ApplicationSerializer.extend({
+  normalizeResponse(store, primaryModelClass, payload, id, requestType) {
+    let totalTokens = payload.data.counters.service_tokens ? payload.data.counters.service_tokens.total : 0;
```

Comment:
i'm not sure we should return 0 here when `service_tokens` doesn't exist, since what could actually be happening is that we're querying for entities or http-requests. what do you think about only attaching the number of tokens to `normalizedPayload` when the `primaryModelClass` is tokens? 

in other words, if we're querying for tokens, update `normalizedPayload` with the number of tokens, if we're querying for entities, update `normalizedPayload` with the number of entities, and so forth. does that make sense?